pocketlint/formatcheck.py

```diff
@@ -460,6 +460,12 @@ def check_regex_line(self, line_no, line):
                     icon='info',
                     )
 
+def check_semantic_newline(self):
+        """Check for a sentence with no new line."""
+        if self.line.find(['. ', '? ', '! ']) != (['\n', '..']):
+            self.message(
```

Comment:
I don't understand what logic is used here.
It helps to add a comment describing what are you targeting here.

looking forward for the tests, as I hope that they can provide more documentation